config/locales/hyrax.en.yml

```diff
@@ -87,6 +87,7 @@ en:
       view_files:               "My Files"
       view_works:               "My Works"
       create_work:              "Deposit Work"
+      describe_work:            "Describe my work →"
```

Comment:
Bridget,
I was wondering about the whitespace/tab (I assume it's a tab) in the quoted part of this line.
Fritz.

Reply:
As pointed out by Bridget, it's actually an arrow character.